src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/endringer/UtledEndringerUtil.kt

```diff
@@ -1,25 +1,135 @@
+import java.time.LocalDate
+
+private typealias PersonendringDetaljerFn<T> = (T, T) -> EndringFelt?
 
 object UtledEndringerUtil {
-    fun finnEndringer(tidligere: PersonopplysningerDto, nye: PersonopplysningerDto) =
-        Endringer(
-            folkeregisterpersonstatus = utledEndringer(tidligere.folkeregisterpersonstatus, nye.folkeregisterpersonstatus),
+
+    fun finnEndringer(tidligere: PersonopplysningerDto, nye: PersonopplysningerDto): Endringer {
+        return Endringer(
+            folkeregisterpersonstatus = utledEndringer(
+                tidligere.folkeregisterpersonstatus,
+                nye.folkeregisterpersonstatus
+            ),
             fødselsdato = utledEndringer(tidligere.fødselsdato, nye.fødselsdato),
             dødsdato = utledEndringer(tidligere.dødsdato, nye.dødsdato),
-            statsborgerskap = utledEndringer(tidligere.statsborgerskap, nye.statsborgerskap),
-            sivilstand = utledEndringer(tidligere.sivilstand, nye.sivilstand),
-            adresse = utledEndringer(tidligere.adresse, nye.adresse),
-            fullmakt = utledEndringer(tidligere.fullmakt, nye.fullmakt),
-            barn = utledEndringer(tidligere.barn, nye.barn), // TODO bedre diff på barn
-            // andreForeldre = // TODO legge til andre forelder og adresse på disse?
-            innflyttingTilNorge = utledEndringer(tidligere.innflyttingTilNorge, nye.innflyttingTilNorge),
-            utflyttingFraNorge = utledEndringer(tidligere.utflyttingFraNorge, nye.utflyttingFraNorge),
-            oppholdstillatelse = utledEndringer(tidligere.oppholdstillatelse, nye.oppholdstillatelse),
-            vergemål = utledEndringer(tidligere.vergemål, nye.vergemål)
+            statsborgerskap = utledEndringerUtenDetaljer(tidligere.statsborgerskap, nye.statsborgerskap),
+            sivilstand = utledEndringerUtenDetaljer(tidligere.sivilstand, nye.sivilstand),
+            adresse = utledEndringerUtenDetaljer(tidligere.adresse, nye.adresse),
+            fullmakt = utledEndringerUtenDetaljer(tidligere.fullmakt, nye.fullmakt),
+            barn = utledEndringerBarn(tidligere.barn, nye.barn),
+            annenForelder = utledEndringerAndreForelder(tidligere.barn, nye.barn),
+            innflyttingTilNorge = utledEndringerUtenDetaljer(tidligere.innflyttingTilNorge, nye.innflyttingTilNorge),
+            utflyttingFraNorge = utledEndringerUtenDetaljer(tidligere.utflyttingFraNorge, nye.utflyttingFraNorge),
+            oppholdstillatelse = utledEndringerUtenDetaljer(tidligere.oppholdstillatelse, nye.oppholdstillatelse),
+            vergemål = utledEndringerUtenDetaljer(tidligere.vergemål, nye.vergemål)
         )
+    }
+
+    private fun <T> utledEndringerUtenDetaljer(
+        tidligere: T,
+        nye: T
+    ) = EndringUtenDetaljer(tidligere != nye)
+
+    private fun <T> utledEndringer(
+        tidligere: T,
+        ny: T,
+        formattertVerdi: (T) -> String = defaultMapper
```

Comment:
Ikke feil, men nyskjerrig. Brukes noe annet enn defaultMapper?  (Eller har vi planer om det?). 
utledEndringer(tidligere.whatever, nye.whatever, enAnnenSpesiellMapper),

Fleksibelt og fint, men krever (av meg 😝) 1 sekund mer å lese.

Reply:
Hadde det først, men ble endret til å legge inn mapping for alle verdier der, så trengs vel ikke :) 
https://github.com/navikt/familie-ef-sak/pull/2055/commits/0714d778dbfb7cbb46355297f2b6b552a126e8da